g4py.sh

```diff
@@ -0,0 +1,51 @@
+package: G4PY
+version: "%(tag_basename)s%(defaults_upper)s"
+tag: v10.3.1
+source: https://github.com/PMunkes/geant4
```

Comment:
This is a patched version of G4PY that was left in here by accident. It should work with the normal GEANT4 repository. I'll change it.